### ISSUE TITLE
💬 Who is using it?

### ISSUE THREAD

**Issue by JohannesKaufmann** [JohannesKaufmann/html-to-markdown#63]
People are using the library for different use cases. Some are using it for better readability of websites, others for migrating content. Knowing the use cases helps to prioritize features and plugins. So I would be interested in... 

➡️ **Who** is using it? Could you let me know **what** you're using the library for? **How** was the experience?

**Comment by lologhi** [comment]
I'm using it with [gocolly/colly](https://github.com/gocolly/colly) to scrap websites (currently, [vatican.va](https://www.vatican.va/content/vatican/en.html)).

**Comment by kevinrobayna** [comment]
Currently using this on my [aoc2md](https://github.com/kevinrobayna/aoc2md) tool!

**Comment by PhiFever** [comment]
Build my own local knowledge database. I wrote some crawlers to save good articles locally from pay-per-month sites like medium.com.

**Comment by ErikKalkoken** [comment]
I am building a service to forward RSS feeds to Discord. Using this nice library to convert the HTML content of RSS feeds to markdown, so I can render them on Discord.

**Comment by K-Sourabh65** [comment]
I’m building a Slack app for a business need that sends notifications to Slack. The input is in HTML format, but Slack supports markdown.

**Comment by JohannesKaufmann** [comment]
@K-Sourabh65 thanks for sharing!

The syntax for Slack is a bit different from markdown. How did that work out? Were there any problems?

For the next versions (v2) I also thought about building a Plugin for that...

**Comment by K-Sourabh65** [comment]
@JohannesKaufmann Yes, I initially faced some issues converting HTML content to Slack's markdown format, but after discovering the custom rule feature in a library, it really helped me.

**Comment by kielmarj** [comment]
> **Who** is using it? Could you let me know **what** you're using the library for? **How** was the experience?

I have been searching for a tool to help me render html nicely within `fzf --preview`, and I'm so happy I've found your work!

I installed 2.2.0 (Linux x86_64) via the releases page in less than 30 seconds and had no trouble using it immediately in various shell scripts. It's working beautifully, thank you! It is faster and has nicer output than everything else I've tried. I like that it is straightforward and doesn't *require* any options or configuration unless desired. Nice work!

Do you have any plans to package `html2markdown` for specific distributions?

**Comment by JohannesKaufmann** [comment]
> render html nicely within `fzf --preview`

@kielmarj Cool program! Thanks for describing your use-case & how the experience was.

>  It's working beautifully, thank you! It is faster and has nicer output than everything else I've tried.

Thanks, that is very nice to hear 😊

> Do you have any plans to package html2markdown for specific distributions?

That would definitely be great! However I don't know much about that (as I'm using MacOS). But I have opened an issue https://github.com/JohannesKaufmann/html-to-markdown/issues/119

**Comment by kielmarj** [comment]
> Cool program! Thanks for describing your use-case & how the experience was.

Here's a link to my [`fzf-pkg-files`](https://github.com/kielmarj/fzf-pkg-files) project that uses `html-to-markdown` for html previews in `fzf`.

> > Do you have any plans to package html2markdown for specific distributions?
> 
> That would definitely be great! However I don't know much about that (as I'm using MacOS). But I have opened an issue #119

I'll check out #119 and see if there's anything I could help with for Linux distributions.

**Comment by bep** [comment]
Hugo just added a template function that uses it: https://gohugo.io/functions/transform/htmltomarkdown/

Works great, simple initial integration.

**Comment by grantcarthew** [comment]
I looked for you on x.com and couldn't find you there, so am dropping a big thanks here.

This package is fantastic. I'm using it in three of my own CLI tools and it made developing them super easy.

So thanks again. Sending love!

Projects:

- [snag](https://github.com/grantcarthew/snag) - Snag web pages like a polite robot with a browser
- [acon](https://github.com/grantcarthew/acon) - Confluence from your terminal - because browser tabs are overrated

The third project is private. It generates resumes and cover letters automatically from job postings.
